modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -1,4 +1,4 @@
-= Provision a Serverless Cluster
+= Create a Serverless Cluster
```

Comment:
I'm curious why it is **Create** here, but the navtree sticks with **Provision**?

Reply:
Just making nav tree and page titles consistent. These sections started with index pages but now we don't have index pages for Dedicated or Serverless.